tests/run/methodmangling_pure_T1382.py

```diff
@@ -0,0 +1,83 @@
+# mode: run
+# tag: pure3.9
```

Comment:
Erm, what was it again that changes in Py3.9?

Reply:
I though I'd set to to a higher number (e.g. `pure100`). I was trying to turn off the "pure" tests for this file since the tests are designed to check that Cython typing is being applied correctly to `__names`. I couldn't find another way to do it. I guess it should (a) actually be higher, and (b) be documented why